src/app/views/Login.svelte

```diff
@@ -185,6 +202,21 @@
         </div>
         <span>Public Key</span>
       </Tile>
+      {#if signerApps.length > 0}
+        {#each signerApps as app}
+          <Tile class="cursor-pointer bg-tinted-800" on:click={() => useSigner(app)}>
+            <div>
+				<img
+                src={`data:image/png;base64,${app.icon}`}
+                alt={app.name}
+                width="48"
+                height="48"
+              />
+            </div>
+            <span>{app.name}</span>
+          </Tile>
+        {/each}
+      {/if}
```

Comment:
This is awesome

Reply:
![Screenshot_20240930_155257](https://github.com/user-attachments/assets/9aa68b7a-2fde-434e-aa31-8d4c42a03e7b)
This is what it looks like on my dev phone with multiple Nip55 signers installed